Mathlib/RingTheory/PowerSeries/Basic.lean

```diff
@@ -1473,6 +1474,7 @@ theorem coeff_zero_one : coeff R 0 (1 : PowerSeries R) = 1 :=
 theorem coeff_mul (n : ℕ) (φ ψ : PowerSeries R) :
     coeff R n (φ * ψ) = ∑ p in Finset.Nat.antidiagonal n, coeff R p.1 φ * coeff R p.2 ψ := by
   symm
+  erw [MvPowerSeries.coeff_mul] -- `rw` can't see that `PowerSeries = MvPowerSeries Unit`
```

Comment:
not to be a nitpicker but in this case shouldn't we just have a lemma to this effect and normal `rw` it? [or does that cause motive issues]

the defeq abuse before wasn't great either though

Reply:
This lemma is the place where we're building that lemma! The problem would go away if we made PowerSeries reducible, or introduced a function to convert between it and MvPowerSeries

Reply:
I'll make a follow-up PR to golf this proof